src/06-errors_and_files/02-except.md

```diff
@@ -13,27 +13,27 @@ readFile :: FilePath -> IO (Either ReadFileError String)
 writeFile :: FilePath -> String -> IO (Either WriteFileError ())
 ```
 
-However there are a couple of issues here, the first is that now composing `IO` actions
-became more difficult. Previously we could write:
+However there are a couple issues here, the first is that composing `IO` actions
+becomes more difficult. Previously we could write:
 
 ```hs
 readFile "input.txt" >>= writeFile "output.html"
 ```
 
-But now the types no longer match - `readFile` will return an `Either ReadFileError String`,
+But now the types no longer match - `readFile` will return an `IO(Either ReadFileError String)`,
 but `writeFile` wants to take a `String` as input. We are forced to handle the error
 before calling `writeFile`.
 
 ## Composing IO + Either using ExceptT
 
-One way to handle this is by using **monad transformers**. Monad transformers are a method
-to stack monad capabilities on top of one another. They are called transformers because
+One way to handle this is by using **monad transformers**, which
+stacks monad capabilities on top of one another. They are called transformers because
```

Comment:
```suggestion
One way to handle this is by using **monad transformers**. Monad transformers provide a way
to stack monad capabilities on top of one another. They are called transformers because
```